CONTRIBUTING.md

```diff
@@ -18,6 +18,23 @@ and/or allows to pool resources early on.
 
 Use GitHub labels as appropriate and feel free to directly add/tag people to the issue.
 
+### Summary of branching model and versioning
+
+Two main branches exist:
+- `master` contains the latest "stable" release
+- `develop` contains all approved changes since the previous release
+
+We aim at creating a new release everyone 4 to 6 weeks.
+The decision on creating a new release is made during the bi-weekly calls.
+A release consists of of merging `develop` into `master` and create a new tag for that version
+using a YY.MM versioning scheme.
```

Comment:
I would prefer something like [semver](https://semver.org/), except start with major version 0 and treat the minor version as the major version until we're ready for a 1.0. So, backwards incompatible changes would result in an update to the minor version, rather than the major version (until we eventually go to version 1.0).

Reply:
Overall sounds good to me, too.
Are you fine with trying to make regular releases (e.g., the 4 to 6 weeks suggested) or have a different suggestion in terms of what "triggers" a new release (major or minor)?

Reply:
I'm fine with 4 to 6 week releases.

Reply:
If you update the language to indicate we're following this slightly-tweaked version of semver, then I'll merge and create the new branches.